src/tests/JIT/Regression/JitBlue/Runtime_105821/Runtime_105821.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Generated by Fuzzlyn v2.2 on 2024-08-01 15:50:23
+// Run on X64 Linux
+// Seed: 16776152078868034128-vectort,vector128,vector256,x86aes,x86avx,x86avx2,x86avx512bw,x86avx512bwvl,x86avx512cd,x86avx512cdvl,x86avx512dq,x86avx512dqvl,x86avx512f,x86avx512fvl,x86avx512fx64,x86bmi1,x86bmi1x64,x86bmi2,x86bmi2x64,x86fma,x86lzcnt,x86lzcntx64,x86pclmulqdq,x86popcnt,x86popcntx64,x86sse,x86ssex64,x86sse2,x86sse2x64,x86sse3,x86sse41,x86sse41x64,x86sse42,x86sse42x64,x86ssse3,x86x86base
+// Reduced from 17.3 KiB to 0.4 KiB in 00:00:34
+// Hits JIT assert in Debug:
+// Assertion failed '(op2->GetRegNum() != targetReg) || (op1Reg == targetReg)' in 'Program:M1(short)' during 'Generate code' (IL size 18; hash 0xb20f93e4; MinOpts)
+// 
+//     File: /__w/1/s/src/coreclr/jit/hwintrinsiccodegenxarch.cpp Line: 2242
+// 
+using System;
+using System.Runtime.CompilerServices;
+using System.Numerics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+public class Program
+{
+    public static void Main()
```

Comment:
Doesn't this need to use xunit, `[Fact]`, and be named something like `TestEntryPoint` to match the convention for tests in this folder?

Presumably `class Program` should then also be named `Runtime_105821` so that we can correctly filter and run it individually?

Reply:
Right, fixed.